djoser/social/token/jwt.py

```diff
@@ -1,9 +1,11 @@
 class TokenStrategy:
     @classmethod
     def obtain(cls, user):
-        from rest_framework_jwt.settings import api_settings
-        payload = api_settings.JWT_PAYLOAD_HANDLER(user)
+        from rest_framework_simplejwt.tokens import RefreshToken
+        from django.utils.six import text_type
+        refresh = RefreshToken.for_user(user)
         return {
-            'token': api_settings.JWT_ENCODE_HANDLER(payload),
-            'user': user
+            'access': text_type(refresh.access_token),
```

Comment:
This breaks all apps that depend on `token` field. We need to keep it. Such changes should be made either using deprecation route or, if radical change is inevitable, a new major version needs to be released - we promised to use semantic versioning.

Reply:
So, this JWT package, which appears to be the only actively maintained one for Django Rest Framework, splits tokens into `access` and `refresh`. Only the `refresh` tokens can be used at the `refresh` endpoint.

I think renaming `access` to `token` would still not be backwards compatible for this reason. Although, token refreshing is broken in django-rest-framework-jwt, so it might not be a big deal.

Ultimately, we need to decide if this is worth shipping a major version. Or, maybe we should take the hint that the community is moving to something else (no actively-maintained JWT packages) and just get rid of JWT completely in favor of a different system.